src/winsign/osslsigncode.py

```diff
@@ -0,0 +1,274 @@
+"""Functions for using osslsigncode utility for signing."""
```

Comment:
Is there a lot of i/o in here? If so, we could look at making these async as well. That probably at least partially depends on whether `osslsigncode` can be called multiple times, concurrently (signing different files).

Reply:
hmmm, good question. it does do a little i/o, but generally is pretty fast. I'll take a look and see if it's easy to add.

Reply:
I tried making these functions async by using asyncio.create_subprocess_exec, but ended up with intermittent test failures I couldn't nail down.

Maybe let's re-visit after if this becomes a problem.

Reply:
+1